integration-tests/src/test/resources/data_stream_api/readonlyrest.yml

```diff
@@ -59,12 +59,56 @@ readonlyrest:
         - cluster:monitor/main
 
     - name: "User6 access streams"
-      auth_key: "user5:pass"
+      auth_key: "user6:pass"
       data_streams:
         - 'data-stream-dev*'
         - 'data-stream-test*'
       actions:
         - indices:admin/data_stream/*
         - indices:monitor/data_stream/*
         - indices:data/read/*
-        - cluster:monitor/main
\ No newline at end of file
+        - cluster:monitor/main
+
+    - name: "User7 access streams"
+      auth_key: "user7:pass"
+      indices:
+        - 'data-stream-dev'
+        - 'data-stream-test'
+      actions:
+        - indices:admin/data_stream/*
```

Comment:
I'm not sure if we should add the actions in our tests. In general, using action is not recommended in ROR. Do we gain sth here? Maybe we can achieve it differently?

Reply:
You are right - we don't need the actions rule here. The one rule per user is enough to be sure that the expected ACL block is tested